webapp/shop/views.py

```diff
@@ -41,7 +41,8 @@ def account_view(**kwargs):
 @use_kwargs(invoice_view, location="query")
 def invoices_view(advantage_mapper: AdvantageMapper, **kwargs):
     marketplace = kwargs.get("marketplace")
-
+    page = kwargs.get("page")
+    rows_per_page = kwargs.get("rows_per_page")
```

Comment:
I would just hardcode this one for now and possibly rename to `per_page`.
`per_page=10`